projects/kit/components/rating/rating.template.html

```diff
@@ -0,0 +1,33 @@
+<div class="t-rating-interactive">
+    <tui-svg
+        *tuiRepeatTimes="let index of max"
+        class="t-rating-icon-filled"
+        [src]="iconFilled"
+        (click)="setRateByIndex(index)"
+    ></tui-svg>
+</div>
+
+<div class="t-rating-wrapper" [style.width.%]="percent">
+    <tui-svg
+        *tuiRepeatTimes="let index of max"
+        class="t-rating-icon-filled"
+        [src]="iconFilled"
+    ></tui-svg>
+</div>
+
+<tui-svg
+    *tuiRepeatTimes="let item of max"
+    class="t-rating-icon-normal"
+    [src]="iconNormal"
+></tui-svg>
+
+<input
+    #focusableElement
```

Comment:
What about meter for disabled/readOnly state?

Reply:
I use host styles for `disabled/readOnly`

![image](https://user-images.githubusercontent.com/12021443/147562830-32558e1b-2cd8-4b7d-ba0a-91141af64595.png)